Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m

```diff
@@ -38,54 +39,114 @@ - (void)setupForTriggerDistance:(CGFloat)overscrollTriggerDistance
-        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:headerFrame];
+        self.headerView = [[VOKDefaultPagingAccessory alloc] init];
+        self.headerView.translatesAutoresizingMaskIntoConstraints = NO;
+        [self.headerView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                                    attribute:NSLayoutAttributeHeight
+                                                                    relatedBy:NSLayoutRelationEqual
+                                                                       toItem:nil
+                                                                    attribute:0
+                                                                   multiplier:1
+                                                                     constant:DefaultAccessoryHeight]];
     }
-    
-    self.headerView.frame = (CGRect){0, -self.headerView.frame.size.height, self.headerView.frame.size};
-    [self.tableView addSubview:self.headerView];
-    
+
+    self.tableView.tableHeaderView = self.headerView;
+
+    [self.tableView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                               attribute:NSLayoutAttributeWidth
+                                                               relatedBy:NSLayoutRelationEqual
+                                                                  toItem:self.tableView
+                                                               attribute:NSLayoutAttributeWidth
+                                                              multiplier:1
+                                                                constant:0]];
+    [self.tableView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                               attribute:NSLayoutAttributeLeft
+                                                               relatedBy:NSLayoutRelationEqual
+                                                                  toItem:self.tableView
+                                                               attribute:NSLayoutAttributeLeft
+                                                              multiplier:1
+                                                                constant:0]];
+    [self.tableView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                               attribute:NSLayoutAttributeTop
+                                                               relatedBy:NSLayoutRelationEqual
+                                                                  toItem:self.tableView
+                                                               attribute:NSLayoutAttributeTop
+                                                              multiplier:1
+                                                                constant:0]];
```

Comment:
Is it necessary to constrain the top-left corner of the header view to the table view's top-left corner?  (Wouldn't that be where the header view is inherently?)